javascript/javascript2/week1/exercise01.js

```diff
@@ -0,0 +1,92 @@
+//Write a function that finds the shortest word of an array of words
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthOfArray = danishWords.length;
+function shortestWords(danishWords) {
+  for (let i = 0; i < lengthOfArray; i++) {
+    let arraysIndexLen = danishWords[i].length;
+    console.log(arraysIndexLen);
+    if (arraysIndexLen <= 1) {
+      return danishWords[i];
+    }
+  }
+}
+console.log(shortestWords(danishWords));
+
+//Find the individual number and the total number of Danish letters in a string.
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+let count = 0;
```

Comment:
Why you need another variable for count total? You can just add all number at the end of the loop to get total.